augur/export_v1.py

```diff
@@ -12,6 +12,7 @@
 from .argparse_ import ExtendOverwriteDefault
 from .errors import AugurError
 from .io.metadata import DEFAULT_DELIMITERS, InvalidDelimiter, read_metadata
+from .io.sequences import read_sequence, read_sequences
```

Comment:
Above, in `augur/align.py`, this line got spelled:

```python
from .io.sequences import read_sequence, read_sequences as io_read_sequences
```

I think I prefer that version to this one, but overarchingly I would like them to both be the same, one way or the other.

Reply:
It's aliased in `align.py` because there is another top-level `read_sequences` function in that file. I don't prefer the `io_` prefix in all other files that use this function. I'll try to come up with a better name for `align.read_sequences` so it doesn't clash with import of `io.read_sequences`.

Reply:
Added 5a4a358a to address this and add a note:

https://github.com/nextstrain/augur/blob/db6fe9f1cce13b8bc7114b131580c3c9209c00f7/augur/align.py#L195-L196